src/appshell/qml/Audacity/AppShell/aboutmodel.cpp

```
@@ -99,3 +332,531 @@ QVariantMap AboutModel::makeUrl(const QUrl& url, bool showPath) const
 
     return urlMap;
 }
+
+QString AboutModel::gplText() const
```

Cppcheck (2.20.0)

[style] The function 'gplText' is never used.

(unusedFunction)